docs/source/platform/schema-validation.md

```diff
@@ -1,60 +1,101 @@
 ---
 title: Validating schema changes
-description: How to validate your schema in your existing CI workflow
+description: How to maintain the schema's contract with CI
 ---
 
-The Apollo GraphQL Platform allows developers to confidently iterate a GraphQL schema by validating the new schema against field-level usage data from the previous schema. By knowing exactly which clients will be broken by a new schema, developers can avoid inadvertently deploying a breaking change.
+To recognize the need for schema validation, first understand that a GraphQL schema defines a contract between clients and server that contains the available types and their behavior. When a GraphQL API is deployed, consumers start to request fields and depend on the contract. If the schema is updated, such as adding a field or removing a type, the contract changes. Modifying the schema and contract can have a wide range of impact on clients from positive, more functionality, to adverse, active schema dependencies no longer exist.
 
-A GraphQL schema can change in a number of ways between releases and, depending on the type of change, can affect clients in a variety of ways. Since changes can range from "decidedly safe" to "certain breakage", it's helpful to use schema tools which are aware of actual API usage.
+The Apollo Platform ensures teams deploy schemas without breaking consumers. To prevent dangerous schema evolution, the `apollo service:check` command compares a proposed schema against the active schema to create a list of changes. This resulting list is matched against the field-level usage data from the previous schema and categorizes the changes by severity. If any change's severity is too high, the team is promptly flagged by the cli or GitHub status with actionable feedback.
```

Comment:
s/cli/CLI? 🤷‍♂️

Reply:
I like Matt's description better, though yours is more accurate. Instead of "This resulting list is matched against the field-level usage data...", it's easier to understand "Engine maintains a list of operations that you've run against that schema & variant, and upon validation, Engine (the apollo platform???) tests that all of those operations still work against the new proposed schema version"